<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QWidget" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ServerWindow</string>
  </property>
  <widget class="QLabel" name="grad">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>491</width>
     <height>531</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>491</width>
     <height>531</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>491</width>
     <height>531</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.263393 rgba(28, 153, 177, 255), stop:1 rgba(8, 0, 179, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="send_text">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="heading">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>491</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.263393 rgba(0, 0, 0, 255));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.263393 rgba(219, 0, 255, 255));</string>
   </property>
   <property name="text">
    <string>                               Client : Agent Beta</string>
   </property>
  </widget>
  <widget class="QLabel" name="you_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 13pt &quot;DejaVu Serif&quot;;
color: rgb(129, 0, 193);
background-color: rgb(0, 0, 0);
border-color: rgb(3, 203, 147);</string>
   </property>
   <property name="text">
    <string>   You&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="send_rec_msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>471</width>
     <height>351</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>471</width>
     <height>351</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>471</width>
     <height>351</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 198, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="snd_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.263393 rgba(180, 255, 209, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.263393 rgba(74, 105, 86, 255));</string>
   </property>
   <property name="text">
    <string>Send &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_client_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(255, 0, 183, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>Start client</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(116, 5, 141);
background-color: rgb(3, 203, 147);</string>
   </property>
   <property name="text">
    <string>                                  Messages</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
